src/lib/utils/getReducedDataUrlFromImage.js

```diff
@@ -0,0 +1,23 @@
+export default (image, maxWidth, maxHeight) => {
```

Comment:
we already have utility that resizes images using canvas
i've linked it above: https://github.com/GoodDollar/GoodDAPP/pull/2016/files#diff-2426bbd8206dd16e2e49bda634acd1edR16

Please
a) move both functions in the single module (e.g. utils/image/resize)
b) move common code (related to creating canvas & resizing) to the private helper function
c) adjust existing functions to use common code. 

As the result you should have module which exports 2 functions - one for resize image form JS Image() class, another one - from base64 data url